### ISSUE TITLE
Deploying solutions

### ISSUE THREAD

**Issue by hosein2398** [yagop/node-telegram-bot-api#301]
Hey guys, this might sound ridiculous to you but I have really many problems on deployment.I tried some servers and now I'm using Heroku. Even after spending hours on it I really don't have a clue on how to deploy my bot on Heroku  (and pls don't say go ahead 'n read Docs and stuff. I've really did my best). All of my projects containing Express work well there , but this is not true about the bot. I deep searched internet on this issue and only found some articles(not really useful) and since Docs are not comprehensive I couldn't get anything out of that.

And I'm just asking you here , How about a small (or better , completed) part in Docs about deployment ?
Do I have to use WebHook while using Heroku to put that to work?
and many other Qs on my mind. I'm really frustrated with this.
Would be thankful if you write a tutorial or something about it.
Thanks.

**Comment by GochoMugo** [comment]
While we admit that our documentation is quite inadequate, we are working towards improving that. We are hoping to receive more contribution in that area particularly.

However, you mentioned that your bot is not working on Heroku. Why is that? Any error? What is the behavior? Could we see the code you using in setting up your bot? These are the kind of questions that might solve your issue (in part).

PS: We could help you write a tutorial in that area as you learn yourself. Talk of killing two birds with one stone!

**Comment by hosein2398** [comment]
I started process of learning how to deploy apps on Heroku the other day and I deployed some Express apps on it with no problem. But about bots I really donnu what to do I donnu I should create WebHook or deploy with Express to put it to work. If you just tell me hints and steps I'll go that way.
And I'll absolutely help to make docs better  when I myself perceived what's going on , put that bird to death ;)
